src/main/resources/sun_checks.xml

```diff
@@ -148,6 +148,7 @@
         <module name="InnerAssignment"/>
         <module name="MagicNumber"/>
         <module name="MissingSwitchDefault"/>
+        <module name="MultipleVariableDeclarations"/>
```

Comment:
As you hit this ground, please update https://checkstyle.org/sun_style.html in the same way as did at https://checkstyle.org/google_style.html , but not all, add line only for point you are try to cover.